plugins/generated.nix

```diff
@@ -0,0 +1,44 @@
+{ pkgs, lib, config, ... }:
+
+with lib;
+
+let
+
+  helpers = import ./helpers.nix { inherit lib config; };
+
+  # names inserted here must match the name of the package in pkgs.vimExtraPlugins
+  plugs = with pkgs.vimExtraPlugins; [
+    "nvim-comment-frame"
+    "vim-printer"
+    "vim-easy-align"
+    "plantuml-syntax"
+    "gruvbox"
```

Comment:
Is there any real reason to have this when we already have the `colorschemes.gruvbox` module? Or maybe add an option to `mkLuaPlugin` that lets you define it as a colorscheme instead of a plugin, so it goes in the `colorschemes` namespace.

Reply:
This is a different gruvbox theme (that I personally like much more then the gruvbox-nvim theme)

Reply:
Probably best to have separate `colorschemes.gruvbox` and `colorschemes.gruvbox-nvim`, imo.

Reply:
Yes, you are right. I will change this